plugins/mcreator-core/blockly/js/mcreator_extensions.js

```diff
@@ -127,6 +127,36 @@ Blockly.Extensions.register('entity_data_string_list_provider',
 
 // Extension used by int providers to validate their min/max values, so that min can't be greater than max and vice versa
 Blockly.Extensions.register('min_max_fields_validator',
+    function () {
+        var minField = this.getField('min');
+        var maxField = this.getField('max');
+
+        if (minField == null) {

```

Comment:
This is quite unclean, revert this change and merge master where I provided the actual fix. In the future, please take time to try and find correct solutions yourself instead of such workarounds, even if it requires another PR before your main PR idea.